shuup/core/cache/impl.py

```diff
@@ -124,7 +129,12 @@ def set(self, key, value, timeout=None, version=None):
             timeout = get_cache_duration(key)
         if version is None:
             version = self.get_version(key)
-        self._cache.set(key, value, timeout=timeout, version=version)
+        try:
+            self._cache.set(key, value, timeout=timeout, version=version)
+        except PicklingError:
+            LOGGER.warning(
```

Comment:
I think you should use `LOGGER.exception` here. This is a critical issue, the developer basically did shit here passing somethings that can't be stored.

Reply:
Yeah, I think that makes most sense!

Reply:
Fixd.